packages/frontend/src/lib/Chip.svelte

```diff
@@ -0,0 +1,16 @@
+<script lang="ts">
```

Comment:
I would call it a badge[^1] if that's okey with you?

Do you think you could also change the following code to use the Chip/badge ?

https://github.com/containers/podman-desktop-extension-ai-lab/blob/af00eed3c16e43ecb721e2b06c33cd3993e30d64/packages/frontend/src/lib/table/model/ModelColumnLabels.svelte#L13

[^1]: https://www.shadcn-svelte.com/docs/components/badge &  https://flowbite.com/docs/components/badge/

Reply:
When i was looking for the name to use i came across https://mui.com/material-ui/react-chip/ , this is why i named like that, but no problem changing it to badge

Reply:
Okey thanks, yeah I guess they both do the same